src/utils/formHelperFunctions.js

```diff
@@ -154,8 +154,9 @@ const nextStepKey = (nextConditions, metadata, fieldValues) => {
         } else {
           if (rule.comparison === 'is_type') {
             ruleMet =
-              (ruleVal === 'email' && emailPattern.test(userVal)) ||
-              (ruleVal === 'phone' && phonePattern.test(userVal));
+              (ruleVal === 'email' && validators.email(userVal)) ||
+              // Not sure if this phone validator can be changed - not the same exact codepath (unlike the email changes)
+              (ruleVal === 'phone' && validators.phone(userVal));
```

Comment:
as called out in the comment above, not sure if this line should be changed. that said, it's not quite clear looking at the code why we'd want to use the phonePattern instead of using `libphonenumber`

Reply:
yeah because here we're just looking to see that it's formatted as a phone number, not that it's an actual valid phone number. that is only needed for explicit error-checking validation logic

Reply:
I kinda feel that it is easier to maintain/simpler if there is just one way